src/styles/bloks/header.scss

```diff
@@ -0,0 +1,65 @@
+.header {
+  font-size: 12px;
+  font-weight: 500;
+
+  display: flex;
+  padding-left: 50px;
+  padding-right: 50px;
+  justify-content: space-between;
+  align-items: center;
+
+  &__logo-link {
+    align-content: center;
+    height: 40px;
+    width: 40px;
+    text-decoration: none;
+  }
+}
+
+.nav {
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style-type: none;
+  }
+
+  &__item {
+    position: relative;
+    margin-right: 20px;
+
+    &--last-link {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    text-decoration: none;
+    align-content: center;
```

Comment:
Similar to the previous comment, `align-content` is not applicable here in `&__link`. If you want to center the text within the link, you might want to use `line-height` property with the same height as the container, or use flexbox properties (`align-items: center`, `justify-content: center`) if the element is a flex container.